src/pages/contributing/documentation.mdx

```diff
@@ -1,71 +1,1382 @@
+Use one image to document keyboard navigation. If the component has limited operation functionality, it may also be incorporated in the same image. Otherwise, the operation should be dealt with in a separate image.
+Normally these images go across 8 columns.
+images should follow the naming convention (component_name-accessibility-1.png
+The ALT text text for the first image can be fairly generic, since the image is just restating information in the prior paragraph.
+The caption should likewise summarize something already described in the preceding paragraph. No new information should be given in the caption. Due to technology constraints, keystroke names in the captions are not given any markdown styling.
+Example:
+![example of breadcrumb keyboard interaction](images/breadcrumb-accessibility-1.png)
+<Caption>
+  The breadcrumb's links are reached by Tab and activated by Enter.
+</Caption>
+-->
+<Row>
+<Column colLg={8}>
+
+![example of (componet_name) keyboard interaction](images/component_name-accessibility-1.png)
+
+<Caption>
+  The (component_name) is reached by Tab and activated by Enter.
+</Caption>
+
+</Column>
+</Row>
+
+<!--
+The Design recommendations section should only be included if annotations are needed
+-->
+
+## Design recommendations
+
+Design annotations are needed for the following instances.
+
+<!--
+Some of the possible subheadings here include:
+- headings
+- alignment
+- labeling
+- meaningful order
+
+Each should have a level 3 heading, a descriptive paragraph and a supporting image.
+
+Example:
+
+### Headings
+Carbon accordions are not set as headings by default. For improved accessibility, annotate accordions as headings on the first occurrence in a product. Annotate the heading level of accordions as needed.
+
+-->
+
+## Development considerations
+
+Keep these considerations in mind if you are modifying Carbon or creating a
+custom component.
+
+<!--
+Provide a bulleted list of implementation considerations for ensuring accessibility. If there is the ability to refernce the IBM toolkit or the ARIA authoring practices, that should be done.
+
+- Use dashed bullets and ensure code is styled as code and consistently appears with the
+following symbols: <element>, 'variable' and "value".
+- End each bulleted list item with a period.
+
+Example:
+- See the [ARIA authoring practices](https://w3c.github.io/aria-practices/#tooltip) for more considerations.
+
+-->
+```
+
+## Documenting patterns
+
+Patterns show reusable combinations of components and templates that address
+common user objectives with sequences and flows. Because they are more
+complicated than components, you may need to adjust the topic order to best tell
+the story — but your pattern should cover all of the topics outlined in the
+following templats.
+
+### Pattern template: for one variant
+
+Use this template for documenting patterns that have a single variant. You can
+see this template in use for [Notification](/patterns/notification-pattern/).
```

Comment:
I would maybe use Text toolbar here as a "one variant" example. This is tricky, because as you said, we really are not consistent with how we treat our anchor links for these patterns. The Notification pattern does have variants or types though which is why I am suggesting a more simple pattern like Text toolbar.